packages/frontend/src/components/common/Filters/configs.ts

```diff
@@ -43,14 +49,43 @@ const timeFilterOptions: Array<{
     { value: FilterOperator.GREATER_THAN_OR_EQUAL, label: 'is on or after' },
 ];
 
+export const getFilterTypeFromField = (field: FilterableField): FilterType => {
```

Comment:
@owlas 
There isn't a good way to know what is the value type of a metric field so I'm defaulting to number.
Should we have a `valueType` for all the fields or at least metrics ? In the case of metrics we can assume the same type as the dimension.
e.g 
```
fieldType: FieldType.METRIC;
type: MetricType.MAX;
valueType: ValueType.DATE;
```